osq-exts/tables/ima/common.go

```diff
@@ -0,0 +1,30 @@
+package ima
+
+import (
+	"bytes"
+	"os"
+)
+
+func pathExists(path string) string {
```

Comment:
osquery-go doesn't have a "bool" type, only int, bigint, double, and String.  Because of this, I had to turn it into a string.
https://github.com/osquery/osquery-go/blob/master/plugin/table/table.go#L152-L157 lists the types supported.